views/includes/annotations/preview.html.swig

```diff
@@ -27,6 +27,16 @@
   {% endif %}
 </div>
 
+{# Plain CSS #}
+{% elseif item.context.type == "css" %}
```

Comment:
Isn't the `<div class="item__code-wrapper"> ... </div>` that other items are using not necessary?